test/tests/p5.Compressor.js

```diff
@@ -0,0 +1,30 @@
+define(['chai'],
+  function(chai) {
+
+  var expect = chai.expect;
+
+  describe('p5.Compressor', function() {
+
+    it('can be created and disposed', function() {
+      var compressor = new p5.Compressor();
+      compressor.dispose();
+    });
+
+     it('wet dry value can be changed', function(){
+      var compressor = new p5.Compressor();
+
+      expect(compressor.drywet(0.5)).to.equal(0.5);
+    });
+
+    it('can set params', function(){
```

Comment:
it might be good to test the `.set` method with incomplete params and ensure it sets the values appropriately